scss/detail.scss

```diff
@@ -15,6 +15,18 @@
   width: 240px;
 }
 
+.redirect {
+  font-size: $text-medium;
+  left: 18%;
+  margin-top: 10%;
+  position: relative;
+  &::before {
+    content: url(./assets/images/back.png);
+    left: -25px;
```

Comment:
don't use negative values. Try using the `right` property instead.

Reply:
ready 👍